app/qml/map/MapWrapper.qml

```diff
@@ -577,6 +584,15 @@ Item {
     }
   }
 
+  NotificationBanner {
```

Comment:
I would say let's leave map out from this one. The `NotificationBanner` can be instantiated in `main.qml`, it does not use anything from map canvas / map elements and in future can be used to display custom messages.

Reply:
So I vote for having the `notificationBanner` in `main.qml`. In that case `function pushNotification` can be inside `NotificationBanner`.